frontend/workflows/projectCatalog/src/details/index.tsx

```diff
@@ -1,39 +1,122 @@
 import React from "react";
-import { Card, Grid, styled } from "@clutch-sh/core";
-import { faGithub, faSlack } from "@fortawesome/free-brands-svg-icons";
+import { useParams } from "react-router-dom";
+import { Grid, styled, Tooltip } from "@clutch-sh/core";
+import { faLock } from "@fortawesome/free-solid-svg-icons";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import GroupIcon from "@material-ui/icons/Group";
+import { capitalize } from "lodash";
+
+import type { DetailWorkflowProps } from "..";
+
+import fetchProject from "./info/resolver";
+import type { ProjectInfo } from "./info/types";
+import type { DetailsCard } from "./card";
+import { MetaCard } from "./card";
+import { ProjectDetailsContext } from "./context";
+import ProjectHeader from "./header";
+import ProjectInfoCard from "./info";
+import QuickLinksCard from "./quick-links";
+
+type CardTyping = React.ReactNode | DetailsCard;
```

Comment:
Wouldn't this be either a meta or dynamic card?

Reply:
Yeah it should be, think this is a hold over from the original implementation. Adjusted